paddle/fluid/inference/tests/api/tester_helper.h

```diff
@@ -351,6 +357,42 @@ void PredictionRun(PaddlePredictor *predictor,
   }
 }
 
+// With support for multiple batches (multiple outputs)
```

Comment:
I wonder why do you rewrite this again, could you enhance line331`predictor->Run(inputs[i], outputs, batch_size);` to `predictor->Run(inputs[i], &(*outputs)[i], FLAGS_batch_size);`

Reply:
The original `PredictionRun` method discards outputs of all but the last iteration and `outputs` is of type `std::vector<PaddleTensor> *`. To calculate average accuracy over all iterations we have to keep all the output data and have `outputs` of type `std::vector<std::vector<PaddleTensor>> *`. I could modify just the original function, but it would require updates also in several other test files. I thought it would be refactored later, but I will do it here if you wish.

Reply:
It's better to refactor here, thanks very much!

Reply:
@luotao1 , I realized that refactoring would require modification of the latency calculation formula. This could influence some latency statistics of other tests. I am not sure whether these statistics are being gathered after running the tests and whether changing them could break anything beyond the tests. I cannot verify that quickly enough so I left the refactoring for later.

Reply:
I wonder why refactoring requires modification of the latency calculation formula?
- Yours:
```
  auto latency = elapsed_time / (iterations * num_times * FLAGS_batch_size);
  PrintTime(FLAGS_batch_size, num_times, FLAGS_paddle_num_threads, 0, latency,
            1);
```
- Ours:
```
PrintTime(batch_size, num_times, num_threads, tid, elapsed_time / num_times,
            inputs.size());
```
Why you don't use ours?

Reply:
With the current implementation fps value would be incorrect. It is 1/latency, so latency is related to a single frame, not a batch. The current formula does not handle that correctly.

Reply:
If current implementation fps value would be incorrect, you can correct it directly.

Reply:
Done.